client/src/utils/CallAutomationUtils.ts

```diff
@@ -0,0 +1,322 @@
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      serverCallId
+    })
+  });
+  if (!response.ok) {
+    console.error('Failed to fetch transcription status:', response);
+    return;
+  }
+  return;
+};
+
+export const startTranscription = async (
+  serverCallId: string,
+  transcriptionOptions?: {
+    locale?: string;
+  }
+): Promise<boolean> => {
+  console.log('Starting transcription for call:', serverCallId);
+  const response = await fetch('/api/startTranscription', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      serverCallId,
+      options: transcriptionOptions
+    })
+  });
+  if (!response.ok) {
+    console.error('Failed to start transcription:', response);
+    return false;
+  }
+  console.log('Started transcription:', transcriptionOptions);
+  return true;
+};
+
+export const stopTranscription = async (serverCallId: string): Promise<boolean> => {
+  console.log('Stopping transcription for call:', serverCallId);
+
+  const response = await fetch('/api/stopTranscription', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      serverCallId
+    })
+  });
+  if (!response.ok) {
+    console.error('Failed to stop transcription:', response);
+    return false;
+  }
+  console.log('Stopped transcription');
+  return true;
+};
+
+export const connectToCallAutomation = async (callAdaterState: CallAdapterState): Promise<boolean> => {
+  if (callAdaterState.call?.info !== undefined && callAdaterState.call?.state === 'Connected') {
+    const serverCallID = await callAdaterState.call.info.getServerCallId();
+    console.log('Server call ID:', serverCallID);
+    const response = await fetch('/api/connectRoomsCall', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        serverCallId: serverCallID
+      })
+    });
+    if (!response.ok) {
+      throw new Error('Failed to start call with transcription');
+    }
+    return true;
+  }
+  return false;
+};
+
+export const getCallSummaryFromServer = async (
+  adapter: CommonCallAdapter,
+  locale: LocaleCode
+): Promise<SummarizeResult> => {
+  console.log('Getting summary from server...');
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const serverCallId = await adapter.getState().call?.info?.getServerCallId();
+    if (!serverCallId) {
+      console.error('Call ID not found');
+      throw new Error('Call ID not found');
+    }
+
+    const response = await fetch('/api/summarizeTranscript', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ serverCallId: serverCallId, locale: locale })
+    });
+    console.log('/summarizeTranscript response', response);
```

Comment:
don't log response, we are logging the summary afterwards. If response needs inspected there's the network tab